Jenkinsfile.benchmark

```diff
@@ -0,0 +1,43 @@
+properties([
+        parameters([
+                string(name: 'BENCHMARKS_FORK', defaultValue: 'PegaSysEng', description: 'The user or org from which to checkout the benchmarks repo', trim: true),
+                string(name: 'BENCHMARKS_BRANCH', defaultValue: 'master', description: 'The benchmarks branch to be checked out', trim: true),
+                string(name: 'DATASET', defaultValue: 'from-0', description: 'The name of the directory containing the benchmark data', trim: true),
+                string(name: 'IMPORT_FILE', defaultValue: '1000.blocks', description: 'The name of the file to import', trim: true)
+        ])
+])
+
+pipeline {
+    agent {
+        docker {
+            image 'pegasyseng/pantheon-build:0.0.3'
+            label 'jenkins-benchmark-1'
+            args '-v /benchmarks/block-import:/benchmarks-block-import-read-only'
+        }
+    }
+
+    stages {
+        stage('Pull pantheon-benchmark') {
+            steps {
+                script {
+                    checkout([$class: 'GitSCM', branches: [[name: "*/${params.BENCHMARKS_BRANCH}"]], doGenerateSubmoduleConfigurations: false, extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'pantheon-benchmarks']], submoduleCfg: [], userRemoteConfigs: [[credentialsId: 'f3f67c22-eead-47db-a8c1-fe32b483583a', url: "git@github.com:${params.BENCHMARKS_FORK}/pantheon-benchmark.git"]]])
+                }
```

Comment:
I updated the benchmarks task to run off of this instance of pantheon rather than the submodule.  That way we should always be running against the right version.  See: https://github.com/PegaSysEng/pantheon/pull/521/files#diff-fa48d841aab37e20f5adc27d017fbc67R32

Reply:
You mean https://github.com/PegaSysEng/pantheon-benchmark/pull/11 - I see that now.  Cool.